### PR TITLE
feat: add support for webworkers

### PR DESCRIPTION
**Pull request by dignifiedquire**
Based on the work in #11  and #12

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,14 +7,14 @@
     "./src/crypto-sha1-2.js": "./src/crypto-sha1-2-browser.js"
   },
   "scripts": {
-    "test": "aegir-test",
-    "test:browser": "aegir-test browser",
+    "test": "aegir-test --webworker",
```
Comment (diasdavid) [inline package.json]: Is this 'just webworker'?

Comment (dignifiedquire) [inline package.json]: yes

File: src/crypto-sha1-2-browser.js [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
+
   if (typeof window !== 'undefined') {
-    if (window.crypto) {
-      return window.crypto.subtle || window.crypto.webkitSubtle
-    }
+    globalContext = window
+  } else if (typeof self !== 'undefined') {
+    globalContext = self
+  } else {
+    return
```
Comment (diasdavid) [inline src/crypto-sha1-2-browser.js]: shouldn't this throw an error?

Comment (dignifiedquire) [inline src/crypto-sha1-2-browser.js]: no, the throwing happens below

File: package.json (hunk 1 above)
Comment (diasdavid) [inline package.json]: So you are proposing to just run the tests in a `webworker`??

Comment (dignifiedquire) [inline package.json]: Ah no, sorry this means run tests also in webworker, that's why it's not `--env webworker`.

Comment (dignifiedquire) [inline package.json]: The tests now run three times, once in node, once in the browser without webworkers and once inside webworkers

Comment (diasdavid) [inline package.json]: aegir could use an update to something like: `--runtime=node, browser, webworker`

Comment (dignifiedquire) [inline package.json]: aegir could use a proper cli interface yes...but don't know when to do things like that...

**Pull request merged**